service/health/response.go

```diff
@@ -17,11 +17,18 @@ type ClusterStatus struct {
 
 // NodeStatus holds data about the health/status of a node.
 type NodeStatus struct {
-	Name         string `json:"name"`
-	Ready        bool   `json:"ready"`
-	IP           string `json:"ip"`
-	Hostname     string `json:"hostname"`
-	InstanceType string `json:"instance_type"`
-	CPUCount     int64  `json:"cpu_count"`
-	MemoryBytes  int64  `json:"available_memory_bytes"`
+	Name                   string `json:"name"`
+	Ready                  bool   `json:"ready"`
+	IP                     string `json:"ip"`
+	Hostname               string `json:"hostname"`
+	InstanceType           string `json:"instance_type"`
+	AvailabilityRegion     string `json:"availability_region"`
+	AvailabilityZone       string `json:"availability_zone"`
+	KubeletVersion         string `json:"kubelet_version"`
+	OperatorVersion        string `json:"operator_version"`
```

Comment:
I can't see this getting filled in the tests, are you sure this is correct?

Reply:
Currently only one AWS test case represents this because the cluster and TC mocks we have are from different clusters, so the node names don't line up. Adding new provider + TC pair mocks is on my todo list